src/main/scala/TestHarness.scala

```diff
@@ -0,0 +1,116 @@
+// See LICENSE for license details.
+
+package rocketchip
+
+import Chisel._
+import cde.{Parameters, Field}
+import rocket.Util._
+import junctions._
+
+class TestHarness(implicit p: Parameters) extends Module {
+  val io = new Bundle {
+    val success = Bool(OUTPUT)
+  }
+  val dut = Module(new Top(p))
+
+  // This test harness isn't especially flexible yet
+  require(dut.io.mem_clk.isEmpty)
+  require(dut.io.mem_rst.isEmpty)
+  require(dut.io.mem_ahb.isEmpty)
+  require(dut.io.mem_tl.isEmpty)
```

Comment:
There are TileLink and AHB test RAMs in uncore.devices. Perhaps we can use those so that we don't have to require mem_ahb and mem_tl be empty. Might be good to move the Nasti RAM there as well.